### PR TITLE
feat(flutter_desktop): colored column names in kanban board

### PR DESCRIPTION
**Pull request by richardshiue**
resolves https://github.com/AppFlowy-IO/AppFlowy/issues/6258

Screenshot 2024-09-11 at 9 03 52 AM

Screenshot 2024-09-11 at 9 04 26 AM

Screenshot 2024-09-11 at 9 05 11 AM

### Feature Preview

---

#### PR Checklist

- [x] My code adheres to [AppFlowy's Conventions](https://docs.appflowy.io/docs/documentation/software-contributions/conventions)
- [x] I've listed at least one issue that this PR fixes in the description above.
- [ ] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [ ] All existing tests are passing.

### CODE REVIEW

File: frontend/appflowy_flutter/pubspec.lock [hunk 1]
```diff
@@ -1559,10 +1559,10 @@ packages:
     dependency: transitive
     description:
       name: platform
-      sha256: "9b71283fc13df574056616011fb138fd3b793ea47cc509c189a6c3fa5f8a1a65"
+      sha256: "12220bb4b65720483f8fa9450b4332347737cf8213dd2840d8b2c823e47243ec"
       url: "https://pub.dev"
     source: hosted
-    version: "3.1.5"
+    version: "3.1.4"
```
Comment (LucasXu0) [inline frontend/appflowy_flutter/pubspec.lock:1565]: I noticed that the versions in pubspec.lock have been changing frequently recently.

The versions of these three packages changed when running pub get. (not sure why)

Comment (Xazin) [reply frontend/appflowy_flutter/pubspec.lock]: What flutter version are you all on?

I'm on `3.22.1` - could be related

Comment (LucasXu0) [reply frontend/appflowy_flutter/pubspec.lock]: I'm using Flutter 3.22.2.

```
Flutter 3.22.2 • channel [user-branch] • unknown source
```

Comment (richardshiue) [reply frontend/appflowy_flutter/pubspec.lock]: 3.22.3 for me. It could be from the newest stable 3.24.x though

**Pull request merged**